msr-flows/flow-tests/complex_serpent_builder.py

```diff
@@ -0,0 +1,245 @@
+# Imports
+import numpy as np
+
+# Constants
+
+
+# Definitions
+def make_input(inp_name, tot_time, time_step, cycle_count):
+    '''
+    This function will generate the input file for Serpent.
+    Naming convention is "name_core_number" for materials.
+    Does not function well for a large number of cycles or time
+    '''
+    lam_cycle = 1
+    setting = 1
+    num_divisions = int(tot_time / time_step)
+    space_between_cores = 110
+    num_cores = cycle_count + 1
+    full_input = '''
+set title "Feedback Run"
+'''
+
+    surface_defs = '''
+%__________SURFACE DEFINITIONS__________%
+'''
+    # Dividing up cores
+    for each_core in range(num_cores):
+        graphite_name = 'ogc_' + str(each_core)
+        inner_fuel_name = 'ifs_' + str(each_core)
+        mult_fac = each_core * space_between_cores
+        x0_fuel = -25 + mult_fac
+        x1_fuel = 25 + mult_fac
+        y0_fuel = -25 + mult_fac
+        y1_fuel = 25 + mult_fac
+        x0_grap = -50 + mult_fac
+        x1_grap = 50 + mult_fac
+        y0_grap = -50 + mult_fac
+        y1_grap = 50 + mult_fac
+        min_z_core = -25
+        max_z_core = 25
+        vol_core = 125000
+        surface_defs += '''
+ % Outer graphite
+ surf {graphite_name} cuboid {x0_grap} {x1_grap} {y0_grap} {y1_grap} -50 50
+ % Inner Fuel Salt Cube vol of 1E6
+ surf {inner_fuel_name} cuboid {x0_fuel} {x1_fuel} {y0_fuel} {y1_fuel} {min_z_core} {max_z_core}
```

Comment:
mmm not sure about this one.
See this https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string
If you can, fix it. If you can't, leave it like that